python/cudf/cudf/io/parquet.py

```diff
@@ -716,6 +721,58 @@ def _get_partitioned(
 
 
 class ParquetDatasetWriter:
+    """
+    Write a parquet file or dataset incrementally
+
+    Parameters
+    ----------
+    path : str or s3 and URL
+        A Local directory path/s3 URL. Will be used as Root Directory
+        path while writing a partitioned dataset.
+    partition_cols : list
+        Column names by which to partition the dataset
+        Columns are partitioned in the order they are given
+    index : bool, default None
+        If ``True``, include the dataframe’s index(es) in the file output.
+        If ``False``, they will not be written to the file. If ``None``,
+        index(es) other than RangeIndex will be saved as columns.
+    compression : {'snappy', None}, default 'snappy'
+        Name of the compression to use. Use ``None`` for no compression.
+    statistics : {'ROWGROUP', 'PAGE', 'NONE'}, default 'ROWGROUP'
+        Level at which column statistics should be included in file.
```

Comment:
Why does compression accept `None` while statistics accepts `'NONE'`? Is this a repeated pattern across cuDF? Can we adopt `None` instead of string `'NONE'` values universally?

Reply:
Yea, I'll open a separate PR to standardize this.